README.md

```diff
@@ -82,6 +82,35 @@ Prior to starting, make sure you have a Babylon GitHub account and that you have
 - [ios-build-distribution](https://github.com/Babylonpartners/ios-build-distribution)
 - [ios-fastlane-match](https://github.com/Babylonpartners/ios-fastlane-match)
 
+On Babylon's Slack, check the channels iOS Team related channels:
```

Comment:
I would re-phrase this to `As an iOS Engineer, you should be in the following Slack channels`.

Reply:
https://github.com/Babylonpartners/ios-playbook/pull/49/commits/2afbc132f6db2fd0b9ef2818b29c8d99c46949c4